### PR TITLE
Linting testing updates

### PR DESCRIPTION
**Pull request by Blackbaud-BrandonJones**

### CODE REVIEW

File: src/help-widget-renderer.ts [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
+import './styles/omnibar-style-adjustments.scss';
+import './styles/widget-styles.scss';
 
 const IFRAME_ID: string = 'bb-help-iframe';
 const IFRAME_TITLE: string = 'BB Help';
 const IFRAME_SRC: string = 'https://host.nxt.blackbaud.com/bb-help/';
-const BB_HEADER_COLOR: string = '#71bf43';
-const BB_HEADER_TEXT_COLOR: string = '#fff';
+const BB_HEADER_COLOR: string = '#71bf43'; // 'rgb(113, 191, 67)';
+const BB_HEADER_TEXT_COLOR: string = '#fff'; // 'rgb(255, 255, 255)';
```
Comment (Blackbaud-TimPepper) [inline src/help-widget-renderer.ts]: Do we need the rgb color values for anything?

Comment (Blackbaud-BrandonJones) [reply src/help-widget-renderer.ts]: Yes unfortunately. Some browsers translate the javascript.style into RGB values.  This is for a reference to those values.

File: src/styles/widget-styles.scss [hunk 2]
```diff
@@ -84,28 +84,27 @@
...
+    text-align: center;
+    width: 2ex;
+    height: 2ex;
+    font-size: 14px;
+    line-height: 2ex;
+    border-radius: 1.4ex;
+    padding: 1px;
+    border: 2px solid;
+    text-decoration: none;
+    box-sizing: content-box!important;
```
Comment (Blackbaud-TimPepper) [inline src/styles/widget-styles.scss]: Can this be accomplished without the `!important` attribute?

Comment (Blackbaud-BrandonJones) [reply src/styles/widget-styles.scss]: Good catch!

**Review by Blackbaud-TimPepper — APPROVED** [review]

**Pull request merged**